app/models/miq_user_role.rb

```diff
@@ -37,8 +37,8 @@ def feature_identifiers
   def allows?(identifier:)
     if feature_identifiers.include?(identifier)
       true
-    elsif parent = MiqProductFeature.parent_for_feature(identifier)
-      allows?(:identifier => parent.identifier)
+    elsif (parent = MiqProductFeature.feature_parent(identifier))
+      allows?(:identifier => parent)
```

Comment:
@chrisarcand aah, ok. This is what I _think_ is happening:

before
-------
parent = `MiqProductFeature` for the parent
`parent.class.name == 'MiqProductFeature'`
pass `parent.identifier` (`class.name == 'String'`) to `allows?`

after
-----
parent = feature identifier for the parent
`parent.class.name == 'String'`
pass `parent` (`class.name == 'String'`) to `allows?`